docker/README.md

```diff
@@ -0,0 +1,27 @@
+# Development auth server
+
+To authenticate users into the Sumé LMS frontend we need first to start an authentication server.
+
+To start it you can use the command bellow in this folder.
+
+```sh
+docker-compose -f docker-compose-dev-auth.yml up
```

Comment:
I think that part is no longer needed

Reply:
This entire Readme at docker folder is useless right now. I gonna remove it. Tks for point this out.